feature/gribi/otg_tests/mpls_in_udp_scale/README.md

```diff
@@ -50,9 +50,24 @@ outer_ip-ttl =        "64"
   * Update policer-policies in a batch of `sched_q` = 1,000
   * Policer-policies changes should take effect within `sched_r` / 2 time
 
-#### Scale profile A - many vlans
+#### Scale profile A - many vlans 1 NH per NHG with same MPLS label
 
 * 20 ip destinations * 1,000 vlans = 20,000 'flows'
+* 1000 VRF have 1000 policy forwarding NHG with 8 NH
```

Comment:
```suggestion
* 1000 network-instances have each have 1 static route pointing a static  NHG with 8 NH.  The encap-headers for the NH should indicate MPLS in GRE encapsulation as per the canonical OC defined in test TE-18.1
```

Reply:
NOte the canonical OC needs to be updated to TE-18.1 to use the static route and static NHG,NH paths defined in https://github.com/openconfig/public/pull/1234